lib/Core/SQLiteBuildDB.cpp

```diff
@@ -435,24 +428,23 @@ class SQLiteBuildDB : public BuildDB {
       }
 
       // Otherwise, read the result contents from the row.
-      assert(sqlite3_column_count(findRuleResultStmt) == 6);
+      assert(sqlite3_column_count(findRuleResultStmt) == 5);
       ruleID = sqlite3_column_int64(findRuleResultStmt, 0);
```

Comment:
Can we call this `dbKeyID`? I think it would be a little more readable below, when we put it into the mapping table.